vendor/manifest

```diff
@@ -77,6 +77,14 @@
 			"notests": true
 		},
 		{
+			"importpath": "github.com/google/uuid",
```

Comment:
So, why the new UUID lib, as opposed to the one we already vendor? This lib doesn't seem to give us anything new.

Reply:
I prefer the Google library :)

> Ok, once you have submitted this pr we can update the request_id package to use the google library also.

Reply:
That's fine, but it's not convincing to me. We already have a UUID library vendored, why must we vendor an entirely new library for just one line of code? Why is the existing lib insufficient?

Reply:
- The project looks like it is abandoned / unmaintained
- Google has a better "Go Report Card" https://goreportcard.com/report/github.com/google/uuid vs https://goreportcard.com/report/github.com/nu7hatch/gouuid
- Google has an active community

....

Reply:
@mholt see this comment: https://github.com/mholt/caddy/pull/1864#discussion_r139295742

Reply:
I have no preference, the nu7hatch one was one I found.  We could pull it out and replace with the google uuid library

Reply:
That's more convincing. Let's replace the nu7hatch lib with the google one. Feel free to make it part of this PR... I hate adding new vendored deps. :P

Reply:
@mholt done 😄